cmd/install/assets/hypershift-operator/hypershift.openshift.io_nodepools.yaml

```diff
@@ -1856,6 +1856,9 @@ spec:
             x-kubernetes-validations:
             - message: Arch is required once set
               rule: '!has(oldSelf.arch) || has(self.arch)'
+            - message: Setting Arch to arm64 is only supported for AWS
+              rule: self.arch != 'arm64' || (has(self.platform.aws) && self.platform.aws.instanceType
```

Comment:
instanceType is a required field so I don't think we need it here, let's drop it unless there's a reason to keep it to avoid any confusion

Reply:
Yeah we do not need it. I tested it without it and all was good.